content/articles/adding-permission-to-flutter-apps/index.md

```diff
@@ -0,0 +1,130 @@
+
+### Adding permission to flutter applications
+### Introduction
+The first important thing to know is that the android system uses permissions. You can for example for your app to be able to use the internet or access the camera.
+So you may not want to let your user's friends download the app on their device if it is not necessary.
+The same, you may not want your user's apps to use your camera if they are not necessary.
+This article is about how to add permission when you create an app with flutter (if you want). It's simple but there are some things to know to avoid.  Your camera usage without permission, for example.
+
+I will write my application so that the user has to accept permissions when he wants them. Users have an option to see all installed apps and select which one can see your camera.
```

Comment:
Use 2nd person singular "You" instead of "I, we, or let", correct this for all instances.